### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#14] [General]
# index

绝对值_x 的博客 快速跳转： 写在前面 往事 20220424 晚，睡前突然萌生了写自己的 blog的想法。当然，这只是个想法而已。但 20220503 晚 我突然间感受到了命运的召唤获得了上手写文的动力。于是，绝对值_x 的博客于 20220504 正式上线。/鼓掌！ 20230712 由于依赖地狱把 node_modules 给崩了，救不回来，于是重新配置博客并更换主题为 VuePress Theme Hope。

https://absx.pages.dev/

**Comment by lilic2233** [comment]
test
